src/Cardano/Prelude/GHC/Heap/NormalForm/Classy.hs

```diff
@@ -43,9 +44,10 @@ import Data.Foldable (toList)
 import Data.Sequence (Seq)
 import Data.List.NonEmpty (NonEmpty)
 import Data.Time
-import GHC.Exts.Heap
+import GHC.Exts.Heap (asBox, getBoxedClosureData)
```

Comment:
Apparently there's a symbol named `info` that's also exported from `GHC.Exts.Heap` that was causing name shadowing in `unsafeNoUnexpectedThunks.errorMessage`. I could have just changed the name of the binding being used in `errorMessage`, but I figured there's no harm in using explicit imports here anyways.

Reply:
Explicit imports for things outside the current repo are always a good idea.